src/fastoad/models/performances/mission/tests/test_routes.py

```diff
@@ -203,174 +82,10 @@ def test_ranged_route(low_speed_polar, high_speed_polar, cleanup):
     )
     flight_points = flight_calculator.compute_from(start)
 
-    # plot_flight(flight_points, "test_ranged_flight.png")
+    # plot_flight(flight_points, "test_ranged_flight.png", RESULTS_FOLDER_PATH)
```

Comment:
Are you willing to keep this comment for latter debug purposes?

Reply:
Yes.
I grant ourselves some more freedom in test code 😄 .